libs/fssr/octree.h

```diff
@@ -183,8 +183,10 @@ class Octree
     int get_num_levels (Node const* node) const;
     void get_samples_per_level (std::vector<std::size_t>* stats,
         Node const* node, std::size_t level) const;
+    /* Pass root node center as parent node center at start of recursion. */
```

Comment:
It seems that this is already the case. The recursion start is [`Octree::influence_query(math::Vec3d const&, double, std::vector<Sample const*>*)`](https://github.com/andre-schulz/mve/blob/1fb6227eb81ede049d96f24e71c37cf5df5a2489/libs/fssr/octree.h#L311) and the recursion itself is [`Octree::influence_query(math::Vec3d const&, double, std::vector<Sample const*>*, Iterator const&, math::Vec3d const&)`](https://github.com/andre-schulz/mve/blob/1fb6227eb81ede049d96f24e71c37cf5df5a2489/libs/fssr/octree.cc#L347).
I'll just remove the comment then.